--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -351,19 +351,21 @@
 	}, {
 		key: 'getEditingArea',
 		value: function getEditingArea() {
-			var element = _reactDom2.default.findDOMNode(this);
-			var editorNode = element.querySelector('.ql-editor');
-			return editorNode;
+			return this.container.querySelector('.ql-editor');
 		}
 	}, {
 		key: 'render',
 		value: function render() {
 			return _react2.default.createElement(
 				'div',
-				{ id: this.props.id,
+				{
+					id: this.props.id,
 					style: this.props.style,
-					className: 'ql-container ' + (this.props.className || '')
-				},
+					className: 'ql-container ' + (this.props.className || ''),
+					onKeyPress: this.onEditorKeyPress,
+					onKeyDown: this.onEditorKeyDown,
+					onKeyUp: this.onEditorKeyUp,
+					ref: (function (el) {this.container = el;}).bind(this)},
 				this.props.children
 			);
 		}
